### DISCUSSION TITLE
Bluetooth 3.0 support?

### DISCUSSION THREAD

**Discussion by guanzhangrtk** [AppGround-io/bluetooth-keyboard-and-mouse-support#149] [Q&A] [answered]
Hi, is there any technical reasons why Bluetooth 3.0 cannot be supported? I have a [NVIDIA Shield portable](https://en.wikipedia.org/wiki/Nvidia_Shield_Portable) and would like to use it to control my iOS devices. Unfortunately it only has Bluetooth 3.0. Thanks for your attention!

**Comment by AppGround-io** [comment, marked as answer]
This is only possible with Android 9.0 and above, earlier versions of Android lack the required APIs.

**Reply by guanzhangrtk** [reply]
I see, thanks for the reply. So the issue is actually Android version. Would it work if I were to update the device to Android 9 or would it still not work due to not having Bluetooth 4.0?

**Reply by AppGround-io** [reply]
It could work when the required API is enabled, because Bluetooth 4.0 is not needed when using Android 9.0.

**Reply by guanzhangrtk** [reply]
Okay thanks for the response. I may install a custom ROM on the SHIELD portable later and see if it works with your app. Thank you again and good luck!
